src/Sulu/Bundle/MediaBundle/Resources/config/services.xml

```diff
@@ -187,8 +187,8 @@
             <tag name="twig.extension"/>
         </service>
 
-        <service id="sulu_media.video_thumbnail_generator" class="%sulu_media.video_thumbnail_generator.class%">
-            <argument type="service" id="dubture_ffmpeg.ffmpeg"/>
+        <service id="sulu_media.video_thumbnail_generator" class="%sulu_media.video_thumbnail_generator.class%" lazy="true">
```

Comment:
Do we need the `lazy` attribute here?

Reply:
same here no idea why this was done - i will take a look at it deeper

Reply:
seems not necessary and is removed.